testgrid/config/config.yaml

```diff
@@ -188,12 +188,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation
 - name: kubernetes-e2e-gce-federation-serial
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-serial
-- name: kubernetes-e2e-gci-gce-federation
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-federation
-- name: kubernetes-e2e-gce-federation-release-1.4
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.4
-- name: kubernetes-e2e-gce-federation-release-1.5
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-federation-release-1.5
+- name: kubernetes-e2e-gce-federation-release-1.6
+  gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-federation-release-1.6
```

Comment:
I'll do a clean up pass, just don't add the entry here, and use `ci-kubernetes-e2e-gce-federation-release-1.6` for now.